### PR TITLE
Introduces exercise 3

### PR DESCRIPTION
**Pull request by vincentaudebert**
@katie to review please 👍

### CODE REVIEW

File: src/components/ExerciseThreeForm.js [hunk 1]
```diff
@@ -7,23 +7,22 @@ const ExerciseThreeForm = () => (
...
-                I need to be a form that takes an input (using InputField component) name
-                and a textarea (using a TextareaField component that you have to create).<br />
-                Label for input is "Your name:" and label for textarea is
-                "Your text:"<br />
-                Please make sure the name is required and raise a warning if it is not only alphabetic string.<br />
-                The textarea field is required, must be alphanumeric and allow spaces, "'" and "." characters. Raise an error if one of these conditions isn't respected.<br />
-                You will raise a warning if the text doesn't contain the sentence 'This is super cool.'<br />
-                Result must show something like that:<br />
-            </p>
+            <p>Create a form that takes an input (using InputField component) with name="name", and a textarea (create your own TextareaField component) with name="text". </p>
```
Comment (katie-day) [inline src/components/ExerciseThreeForm.js]: I've just added some formatting and modified the copy a bit, I just found it a little hard to read and follow the steps I was too take, so I just split it up so its a little easier to follow :)

File: src/index.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 body {
   margin: 10px;
   padding: 0;
-  width: 750px;
+  max-width: 750px;
```
Comment (katie-day) [inline src/index.css:4]: Added `max-` so it's responsive ;)

Comment (vincentaudebert) [reply src/index.css:4]: Yeah I made that change into master already. It shouldn't be a problem at merging.

File: src/components/ExerciseThreeForm.js (hunk 1 above)
Comment (vincentaudebert) [reply src/components/ExerciseThreeForm.js]: Brilliant! Thanks a lot Katie!

**Pull request merged**